doc/source/deployment.rst

```diff
@@ -38,13 +38,21 @@ Deployment of the complete system in a single machine/VM.
 
 #. Generate the SSL certificates if you do not already have them. The
    resulting certificates will have names test.* because they are
-   self-signed and are not supposed to be used for production.  Once
-   created, copy them to the jupyterhub directory::
+   self-signed and are not supposed to be used for production.
```

Comment:
As the paragraph is getting a bit long, could you bold this line and make sure people read it please?